### DISCUSSION TITLE
Zoom on image part

### DISCUSSION THREAD

**Discussion by AIrtistry** [Mikubill/sd-webui-controlnet#1829] [Q&A] [answered]
Hello guy's, Is it possible to zoom the controlnet image menu for make accurate edits for inpainting for example ? The window is very small and the only solution for me is to scroll the whole a1111 ui, I try to install the zoom extension but nothing showing on the controlnet tab.

Thanks you

**Comment by huchenlei** [comment, marked as answer]
See https://github.com/Mikubill/sd-webui-controlnet/issues/1504

**Reply by huchenlei** [reply]
Hit s on input image should bring up the zoomed view.

**Reply by AIrtistry** [reply]
thanks
